be/src/olap/olap_index.h

```diff
@@ -87,13 +83,11 @@ struct OLAPIndexOffset {
 
 // 唯一标识一个RowBlock在Data文件和Index文件中的位置
 struct RowBlockPosition {
-    RowBlockPosition() : segment(0), block_size(0), data_offset(0), index_offset(0) {}
+    RowBlockPosition() : segment(0), data_offset(0) {}
```

Comment:
I think block_index is a better name for data_offset

Reply:
this is used by many other classes which may be conflict with be_refactor.
We can modify it later